source/atlas-cli-deploy-docker.txt

```diff
@@ -37,43 +37,40 @@ deployment with Docker.
 
       To learn more, see the `Docker documentation <https://docs.docker.com/desktop/install/mac-install/>`__.
 
-   .. step:: Pull down the latest ``mongodb/atlas`` Docker image.
-
-      **Example:**
+   .. step:: Pull down the latest ``mongodb/mongodb-atlas-local`` Docker image.
```

Comment:
the text `Use the atlas deployments command to create a local deployment with Docker.` is no longer relevant, we don't need `atlas deployments` anymore

Reply:
`:latest` is optional for any docker image, but we do offer MongoDB versions like `docker pull mongodb/mongodb-atlas-local:7.0.4` and also `docker pull mongodb/mongodb-atlas-local:7.0` or `docker pull mongodb/mongodb-atlas-local:latest`